angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/search-results.html

```diff
@@ -0,0 +1,76 @@
+<div class="row portlet-frame col-xs-12 col-md-12 no-padding search-results" ng-controller="MarketplaceController as marketplaceCtrl">
+	<div class="inner-nav-container">
+		<ul class="inner-nav">
+			<li class="active">
+				<a>All Results ({{ results.length }})</a>
+			</li>
+			<!-- <li>
+				<a>MyUW (# results)</a>
+			</li>
+			<li>
+				<a>Directory (# results)</a>
+			</li>
+			<li>
+				<a>Wisc.edu (# results)</a>
+			</li> -->
+		</ul>
+	</div>
+
+
+  <!-- <loading-gif data-object='portlets'></loading-gif> -->
```

Comment:
The initial load will still be the 7 seconds in prod (or whatever it is). Do we want this loading state?